src/spikeinterface/postprocessing/correlograms.py

```diff
@@ -107,6 +128,33 @@ def compute_correlograms(
 
 
 def _make_bins(sorting, window_ms, bin_ms):
+    """
+    Create the bins for the correlogram, in samples.
+
+    The autocorrelogram bins are centered around zero. Each bin
+    increases in a positive / negative direction starting at zero.
+
+    For example, given a window_ms of 50 ms and a bin_ms of
+    5 ms, the bins in unit ms will be:
+    [-25 to -20, ..., -5 to 0, 0 to 5, ..., 20 to 25].
+
+    The window size will be clipped if not divisible by the bin size.
+
+    Parameters
+    ----------
+    See ComputeCorrelograms() for parameters.
+
+    Returns
+    -------
+
+    bins : np.ndarray
+        The bins edges in ms
+    window_size : int
+        The window size in samples
```

Comment:
One thing we should discuss is that we have `frame_slice` and now we are saying `samples`. I personally prefer sample terminology, but what do thing about frames/samples vs mixed terminology vs just using frames since that is already built deeply into the library and then we explain the equivalence in the future magic glossary?

Reply:
I also prefer sample.
frame is totally an historical stuff done by Alessio or Jeremy I guess.
If day we go to version one we need to stop using "frame" I guess but Alessio will be unhappy with this.

Reply:
This is interesting,I am not very familiar with this [distinction between samples and frames](https://www.mathworks.com/help/dsp/ug/sample-and-frame-based-concepts.html#:~:text=and%20Frame%20Rate-,Samples%20and%20Frames,of%20consecutive%20times%20stacked%20together.). I did not know this difference until I just looked it up. If I have understood correctly, I _think_ in this instances 'samples' would be correct, for example we have a window size of 10 samples (that together would make up 1 frame). If you had a window size of 10 frames I think this would imply 10 sets of timeseries? It makes sense for `frame_index` I guess as it reads 'index of the frame you want to slice'? Although I also prefer sample as I think `frame` is less well known.

Reply:
I've linked to this #2800 for futher discussion